server/src/test/java/org/elasticsearch/indices/analysis/lucene/SkipAsciiDigits.java

```diff
@@ -6,24 +6,26 @@
  * Side Public License, v 1.
  */
 
-package org.elasticsearch.example.analysis.lucene;
+package org.elasticsearch.indices.analysis.lucene;
 
 import org.apache.lucene.analysis.FilteringTokenFilter;
 import org.apache.lucene.analysis.TokenStream;
 import org.apache.lucene.analysis.tokenattributes.CharTermAttribute;
 
 import java.io.IOException;
 
-public class Skip1TokenFilter extends FilteringTokenFilter {
+public class SkipAsciiDigits extends FilteringTokenFilter {
```

Comment:
this looks confusing because the same classes are used in server for unit testing. 
`Skip1TokenFilter` in examples was replaced with `SkipStartingWithDigitTokenFilter`
I will rename the classes in server to reflect those in examples.